packages/react/fast-components-styles-msft/src/utilities/color/accent-fill-gradient.ts

```diff
@@ -0,0 +1,55 @@
+import { DesignSystem } from "src";
+import { ColorHSL, hslToRGB, parseColor, rgbToHSL } from "@microsoft/fast-colors";
+import { format } from "@microsoft/fast-jss-utilities";
+import {
+    FillSwatchFamily,
+    Swatch,
+    SwatchFamily,
+    SwatchFamilyResolver,
+    swatchFamilyToSwatchRecipeFactory,
+    SwatchFamilyType,
+    SwatchRecipe,
+    SwatchResolver,
+} from "./common";
+import { accentFill } from ".";
+
+function gradientSwatchRecipe<T extends SwatchFamily>(
+    type: keyof T,
+    callback: SwatchFamilyResolver<T>
+): SwatchResolver {
+    return (designSystem: DesignSystem): Swatch => {
+        const baseRecipe: SwatchRecipe = swatchFamilyToSwatchRecipeFactory<T>(
+            type,
+            callback
+        );
+        const color: Swatch = baseRecipe(designSystem);
```

Comment:
Could this be `minColor` instead of `color`? (see my note below)

Reply:
Made it baseColor.